build.gradle

```diff
@@ -55,7 +55,7 @@ repositories {
 
 }
 
-final htsjdkVersion = System.getProperty('htsjdk.version','2.8.1-22-gb05d1f2-SNAPSHOT')
+final htsjdkVersion = System.getProperty('htsjdk.version','2.9.0-1-g55bf01b-20170217.164935-1')
```

Comment:
Is there some difference between using this name and using 2.9.0-1-g55bf01b-SNAPSHOT ? Otherwise this looks fine - just wondering if there is a reason for this.